front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineTemplateController.java

```diff
@@ -37,29 +37,42 @@
 @RequestMapping("pipelineTemplates")
 public class PipelineTemplateController {
 
-  @Autowired
-  PipelineTemplateDAO pipelineTemplateDAO;
+  @Autowired(required = false)
+  PipelineTemplateDAO pipelineTemplateDAO = null;
 
   // TODO rz - Add fiat authz
 
   @RequestMapping(value = "", method = RequestMethod.GET)
   List<PipelineTemplate> list(@RequestParam(required = false, value = "scopes", defaultValue = "global") List<String> scopes) {
+    if (pipelineTemplateDAO == null) {
+      throw PipelineTemplateFeatureUnsupportedException.build();
+    }
     return (List<PipelineTemplate>) pipelineTemplateDAO.getPipelineTemplatesByScope(scopes);
```

Comment:
could cut down the duplicated code w/ ...

```
PipelineTemplateDAO getPipelineTemplateDAO() {
   if (pipelineTemplateDAO == null) {
      throw exception;
   }

   return pipelineTemplateDAO;
}
```

and then just use `getPipelineTemplateDAO()` in each method.